templates/taxi/driver_list.html

```diff
@@ -6,8 +6,12 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.id == user.id %}
```

Comment:
You can add condition only for '(Me)' to avoid writing a lot of code

Reply:
Got it, thanks)